app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car) -> list:
+        price = car.comfort_class * (self.clean_power - car.clean_mark)
+        result_price = price * self.average_rating / self.distance_from_city_center
+        return round(result_price, 1)
+
+    def wash_single_car(self, car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rate) -> None:
+        result = self.average_rating * self.count_of_ratings
+        self.count_of_ratings += 1
+        result += rate
+        self.average_rating = round(result / self.count_of_ratings, 1)
+
+    def serve_cars(self, cars) -> int:
+        result = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                result += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return result
```

Comment:
The `serve_cars` method calculates the total washing price for all cars that need washing. Ensure that the logic aligns with the task requirements, especially if there are specific conditions or calculations that need to be applied to each car.